src/targets/gpu/pack_int8_args.cpp

```diff
@@ -69,6 +93,76 @@ shape pack_int8_args::pack_int8_shape(const shape& s) const
     return {s.type(), lens, strides};
 }
 
+instruction_ref pack_int8_args::pad_ins(module& p, instruction_ref ins, int offset) const
```

Comment:
This can be a free function in the cpp file.

Reply:
changed